tests/testthat/test-renderer3-HitOrMissMonteCarlo.R

```diff
@@ -0,0 +1,171 @@
+chunk3 <- read.table(chunk3.tsv, sep="\t", header=TRUE,
+                     comment.char="", quote="")
+
+chunk4.tsv <- file.path("animint-htmltest", "geom6_text_viztwo_chunk1.tsv")
+chunk4 <- read.table(chunk4.tsv, sep="\t", header=TRUE,
+                     comment.char="", quote="")
+
+chunk5.tsv <- file.path("animint-htmltest", "geom4_hline_viztwo_chunk1.tsv")
+chunk5 <- read.table(chunk5.tsv, sep="\t", header=TRUE,
+                     comment.char="", quote="")
+
+chunk6.tsv <- file.path("animint-htmltest", "geom3_tallrect_viztwo_chunk1.tsv")
+chunk6 <- read.table(chunk6.tsv, sep="\t", header=TRUE,
+                     comment.char="", quote="")
+
+test_that("Integration Curve data contains expected columns", {
+  path_expected_cols <- c("x", "y", "group")
+  point_expected_cols <- c("shape", "colour", "x", "y", "key", "showSelectedlegendshapecolour", "showSelected1", "fill")
+  
+  expect_identical(sort(names(chunk1)), sort(path_expected_cols))
+  expect_identical(sort(names(chunk2)), sort(point_expected_cols))
+})
+
+test_that("Pi estimation graph data contains expected columns", {
+  path_expected_cols <- c("x", "y", "showSelected1", "key", "group")
+  text_expected_cols <- c("x", "y", "key", "label", "showSelected1", "group")
+  tallrect_expected_cols <- c("xmin", "xmax", "clickSelects", "group")
+  hline_expected_cols <- c("yintercept")
+  
+  expect_identical(sort(names(chunk3)), sort(path_expected_cols))
+  expect_identical(sort(names(chunk4)), sort(text_expected_cols))
+  expect_identical(sort(names(chunk5)), sort(hline_expected_cols))
+  expect_identical(sort(names(chunk6)), sort(tallrect_expected_cols))
+})
+
+test_that("x and y labels are not null", {
+  xlabel <- getNodeSet(info$html, "//text[@class='xtitle']")
+  ylabel <- getNodeSet(info$html, "//text[@class='ytitle']")
+  
+  expect_equal(length(xlabel), 2)
+  expect_equal(length(ylabel), 2)
+})
+
+test_that("Different Points Are Rendered on Integration Curve", {
+  x1_nodes <- getNodeSet(info$html, "//g[@class='geom2_point_vizone']/g[@class='PANEL1']/circle/@cx")
+  y1_nodes <- getNodeSet(info$html, "//g[@class='geom2_point_vizone']/g[@class='PANEL1']/circle/@cy")
+  
+  Sys.sleep(1.739)
+
+  info$newHtml <- getHTML()
+
+  x2_nodes <- getNodeSet(info$newHtml, "//g[@class='geom2_point_vizone']/g[@class='PANEL1']/circle/@cx")
+  y2_nodes <- getNodeSet(info$newHtml, "//g[@class='geom2_point_vizone']/g[@class='PANEL1']/circle/@cy")
+
+  expect_false(identical(x1_nodes, x2_nodes))
+  expect_false(identical(y1_nodes, y2_nodes))
+})
+
+test_that("Different points are rendered on Pi estimation plot", {
+  coordPanel1 <- getNodeSet(info$html, "//g[@class='geom5_path_viztwo']/g[@class='PANEL1']/path/@d")
+
+  Sys.sleep(1.739)
+
+  info$newHtml <- getHTML()
+
+  coordPanel2 <- getNodeSet(info$newHtml, "//g[@class='geom5_path_viztwo']/g[@class='PANEL1']/path/@d")
+
+  expect_false(identical(coordPanel1, coordPanel2))
```

Comment:
Does this test assume that `getNodeSet` will always return the rendered points in the same order? I would prefer not to check it this way because even if the plot is not rendering any new points if getNodeSet returns them in a different order, this test might pass. Though if you are sure about the ordering, no changes needed. To me, the length of the retrieved nodes is a better indicator.

Reply:
I guess checking the length of retrieved nodes is a better indicator. I'll make changes and push as soon as possible.

Reply:
Hii @tdhock, @faizan-khan-iit, 
I observed that the points are being rendered in order. 
I checked and found this about the order.
![Screenshot from 2020-04-14 13-29-29](https://user-images.githubusercontent.com/17141301/79200526-3634bd00-7e54-11ea-8121-7a4adf588f0a.png)
![Screenshot from 2020-04-14 13-30-17](https://user-images.githubusercontent.com/17141301/79200535-38971700-7e54-11ea-8c10-54e4def8012c.png)

So, the identical check is good to go. But if you say I can move to length check which will also work!